roles/db-copy/tasks/active-copy.yml

```diff
@@ -0,0 +1,137 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+- name: Active-copy | Check pmon process
+  shell: "set -o pipefail; ps -ef | ( grep pmon || true ) | ( grep -i {{ db_name }} || true ) | ( grep -v grep || true ) | wc -l"
```

Comment:
Can we be a bot more specific about the instance name to avoid matching prefixed or suffixed SIDs?  I think we want to see pmon_(db name)(node number if RAC), or as a regex, pmon_{{db_name}}[0-9]?

Reply:
there is {{ oracle_sid }} which I changed db_name to

Reply:
I think we have the same problem here:  if you have a database called db1 and one called db10, we'll match both, and even maybe match arguments.  In this case I think we want a process called ora_pmon_{{ db_name }} and an optional RAC instance suffix.

For a DB called orcl, pgrep might do this in a cleaner way

pgrep 'ora_pmon_orcl[0-9]?$'

Or, even better, it might be possible to do in pure ansible

https://docs.ansible.com/ansible/latest/modules/pids_module.html

Reply:
I added $ to {{ oracle_sid }} to catch it properly